tests/components/lastfm/test_sensor.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for the lastfm sensor."""
+from unittest.mock import patch
+
+from pylast import Track
+import pytest
+
+from homeassistant.components import sensor
+from homeassistant.components.lastfm.sensor import STATE_NOT_SCROBBLING
+from homeassistant.setup import async_setup_component
+
+
+class MockUser:
+    """Mock user object for pylast."""
+
+    def __init__(self, now_playing_result):
+        """Initialize the mock."""
+        self._now_playing_result = now_playing_result
+
+    def get_playcount(self):
+        """Get mock play count."""
+        return 1
+
+    def get_image(self):
+        """Get mock play count."""
+        pass
+
+    def get_recent_tracks(self, limit):
+        """Get mock recent tracks."""
+        return []
+
+    def get_top_tracks(self, limit):
+        """Get mock top tracks."""
+        return []
+
+    def get_now_playing(self):
+        """Get mock now playing."""
+        return self._now_playing_result
+
+
+@pytest.fixture(name="get_user")
+def lastfm_get_user_fixture():
+    """Create fixture for get_user."""
+    with patch("pylast.LastFMNetwork.get_user") as get_user:
```

Comment:
I think we need to patch the network class too, as we discussed. There seems to be side effects when that class is instantiated:
https://github.com/pylast/pylast/blob/a040f42f9570c1a5edb77cdebded35c019349edf/src/pylast/__init__.py#L191-L205

Reply:
What would you patch here?

Reply:
See my latest commit - I followed the same approach but didnt actually do anything since I dont need to.